### PR TITLE
Add support for populating data from the flashcard database

### PR DESCRIPTION
**Pull request by Cheng20010201**
Add functionality for reading data from the database created and populate it into the application when the application first starts.

Addition library simple-json is used and put in the **Acknowledgements** section of `README.md`.
- The reason that jacson is not used is that simple json is more convenient as we do not need a serializable object, but only need to read json data and parse the content.

### CODE REVIEW

**Review by Jellybeano — APPROVED** [review]
igtm~

File: src/main/java/seedu/address/model/util/LocalDatabasePopulator.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public static Set<Tag> getTagSet(String... strings) {
+        return Arrays.stream(strings)
+                .map(Tag::new)
+                .collect(Collectors.toSet());
+    }
+
+    /**
+     * Returns a tag set containing the tags contained in the json flashcard object.
+     */
+    public static Set<Tag> getTagSet(JSONArray jsonTags) {
```
Comment (xinweit) [inline src/main/java/seedu/address/model/util/LocalDatabasePopulator.java:64]: lgtm. just want to clarify the JSONArray jsonTags passed as a parameter here is a json array of all of the 'tagged' field values in our database flashcards.json?
Screenshot 2021-03-09 135738

**Review by xinweit — APPROVED** [review]
lgtm

File: src/main/java/seedu/address/model/util/LocalDatabasePopulator.java (hunk 1 above)
Comment (Cheng20010201) [reply src/main/java/seedu/address/model/util/LocalDatabasePopulator.java:64]: should be json array of all tags of one single flashcard

**Pull request merged**